ciongke/sinker_deployment.yaml

```diff
@@ -14,4 +14,3 @@ spec:
       containers:
       - name: sinker
         image: gcr.io/kubernetes-jenkins-pull/sinker:0.0
-      restartPolicy: Never
```

Comment:
I guess I should have said something about it, sorry. I kind of wondered why you had done that :-)